tests/behat/uploadFileToMoodle.feature

```diff
@@ -43,3 +43,33 @@ Feature: upload the resource in oCIS to moodle
     And I click on "//*[@class='fp-filename-field']/p[text()='testfile.txt']" "xpath_element"
     And I click on "Select this file" "button"
     Then I should see "testfile.txt"
+
+  Scenario: click refresh button to get latest resource on Personal drive
+    Given I click on "Personal" "link"
+    And user "admin" has uploaded a file inside space "Personal" with content "some content" to "testfile.txt"
+    And I should not see "testfile.txt"
+    When I refresh the file-picker
+
+  Scenario: click refresh button to get latest resource on Project drive
+    Given "admin" has created the project space "ProjectMoodle"
+    And I should not see "ProjectMoodle"
+    When I refresh the file-picker
+    Then I should see "ProjectMoodle"
+    And I click on "ProjectMoodle" "link"
+    And user "admin" has uploaded a file inside space "ProjectMoodle" with content "some content" to "testfile.txt"
+    And I should not see "testfile.txt"
+    When I refresh the file-picker
+    Then I should see "testfile.txt"
+
+  Scenario: click refresh button to get latest resource of Share drive
+    Given I click on "Shares" "link"
+    And user "Brian" has been created with default attributes
+    And user "Brian" has uploaded a file inside space "Personal" with content "some content" to "/testfile.txt"
+    And user "Brian" has sent the following share invitation:
+      | resource        | testfile.txt |
+      | space           | Personal     |
+      | sharee          | Admin        |
+      | shareType       | user         |
+    And I should not see "testfile.txt"
+    When I refresh the file-picker
+    Then I should see "testfile.txt"
```

Comment:
Note: I think that it is OK to have these alternating When and Then steps for this sort of test scenario.
The thing that we are testing is that the resource is not seen immediately, but is seen after a refresh. So a refresh has to be done between the 2 Then steps.

We could have a step like:
```suggestion
    But I should see "testfile.txt" after refreshing the file-picker
```

And that step will do the refresh and the check.
I don't really know if that is better.